src/Profiling.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
```

Comment:
Does this always work in all C++ environments? For consistency, let's just use macros.
```suggestion
#ifndef HPGMP_PROFILING_HPP
#define HPGMP_PROFILING_HPP
```

Reply:
Addressed in [a3d00b8](https://github.com/at-aaims/HPG-MxP/pull/3/commits/a3d00b811a5f9cee298f4b3b803a2543061aabdf)